pkg/uptimerobot.go

```diff
@@ -215,11 +223,34 @@ func (c *Client) GetMonitorByID(ID int64) (Monitor, error) {
 
 // GetMonitors returns a slice of Monitors representing the existing monitors.
 func (c *Client) GetMonitors() (monitors []Monitor, err error) {
```

Comment:
Could you add a test for the new functionality too? It doesn't matter if it's the most basic and hacky of tests; I'm going to refactor and improve the tests wholesale as soon as I can.

Reply:
Just need to add the tests now